npm-shrinkwrap.json

```diff
@@ -6,7 +6,7 @@
       "version": "1.0.3"
     },
     "abbrev": {
-      "version": "1.0.7"
+      "version": "1.0.9"
```

Comment:
>Why is shrinkwrap changing versions on modules we haven't touched?

As I understand it, this is intentional, and _may_ be related to npm 3 dependency resolution ([docs](https://github.com/Automattic/wp-calypso/blob/master/docs/shrinkwrap.md#npm-3), cc @gwwar ).